DEVELOPER.md

```
@@ -0,0 +1,108 @@
+2.  **Install dependencies:**
+    ```bash
+    pnpm install
+    ```
+    This will install all project dependencies, including development and peer dependencies.
+
+## Project Structure
+
+-   `src/`: Contains the source code for the Angular wrapper component.
+    -   `index.ts`: The main entry point for the library, exporting `InteractiveVideoComponent` and related types.
+-   `test/`: Contains unit tests for the components.
+    -   `interactive-video.test.ts`: Tests for the `InteractiveVideoComponent`.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)